docker/1.6.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,112 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.6.8
+ARG MX_URL=https://aws-mxnet-pypi.s3-us-west-2.amazonaws.com/1.6.0/aws_mxnet_mkl-1.6.0rc0-py2.py3-none-manylinux1_x86_64.whl
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV SAGEMAKER_TRAINING_MODULE sagemaker_mxnet_container.training:main
+ENV DGLBACKEND=mxnet
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    software-properties-common \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    libopencv-dev \
+    openssh-client \
+    openssh-server \
+    vim \
+    wget \
+    zlib1g-dev \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
+ && tar -xvf Python-$PYTHON_VERSION.tgz \
+ && cd Python-$PYTHON_VERSION \
+ && ./configure \
+ && make \
+ && make install \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends \
+    libreadline-gplv2-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libsqlite3-dev \
+    libgdbm-dev \
+    libc6-dev \
+    libbz2-dev \
+    tk-dev \
+ && make \
+ && make install \
+ && rm -rf ../Python-$PYTHON_VERSION* \
+ && ln -s /usr/local/bin/pip3 /usr/bin/pip \
+ && ln -s $(which ${PYTHON}) /usr/local/bin/python \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
+
+WORKDIR /
+
+# Docker container build should happen in a fresh environment with context for this build.
```

Comment:
Remove

Reply:
this line and the line below? in all dockerfiles?